httpclient5/src/main/java/org/apache/hc/client5/http/entity/mime/FormBodyPartBuilder.java

```diff
@@ -47,22 +50,52 @@ public class FormBodyPartBuilder {
     private ContentBody body;
     private final Header header;
 
+    /**
+     * The multipart mode determining how filenames are encoded in the {@code Content-Disposition}
+     * header, defaults to {@link HttpMultipartMode#STRICT}.
+     *
+     * @since 5.5
+     */
+    private HttpMultipartMode mode;
+
+    /**
+     * Reusable encoder for ISO-8859-1 charset checks, improving performance by avoiding
+     * repeated instance creation.
+     */
+    private final CharsetEncoder iso8859_1Encoder = StandardCharsets.ISO_8859_1.newEncoder();
```

Comment:
@arturobernalg I would instantiate this attribute lazily. It does not need to be final. Looks good overall.

Reply:
@ok2c changed.